src/pattern.jl

```diff
@@ -15,7 +15,7 @@ Supports [`ConnectivityTracer`](@ref), [`GradientTracer`](@ref) and [`HessianTra
 """
 trace_input(::Type{T}, x) where {T<:AbstractTracer} = trace_input(T, x, 1)
 
-function trace_input(::Type{T}, x::Number, i::Integer) where {T<:AbstractTracer}
+function trace_input(::Type{T}, x::Real, i::Integer) where {T<:AbstractTracer}
```

Comment:
Needs support for returning a Complex{T} when called with Complex number.

Reply:
#102